lte/gateway/c/core/oai/lib/s6a_proxy/proto_msg_to_itti_msg.cpp

```diff
@@ -134,6 +134,20 @@ void convert_proto_msg_to_itti_s6a_update_location_ans(
     itti_msg->subscription_data.access_mode = NAM_ONLY_PACKET;
   }
 
+  // Regional subscription zone codes
+  itti_msg->subscription_data.num_zcs =
+      msg.regional_subscription_zone_code_size();
+  uint8_t itr = 0;
+  while (itr < itti_msg->subscription_data.num_zcs) {
+    auto regional_subscription_zone_code =
+        msg.regional_subscription_zone_code(itr);
+    memcpy(
+        itti_msg->subscription_data.reg_sub[itr].zone_code,
+        regional_subscription_zone_code.c_str(),
+        regional_subscription_zone_code.length());
```

Comment:
how do you ensure that zone_code has large enough buffer? We should truncate or ignore if the zone code is too long.

Reply:
How do we know if its too long? its a string

Reply:
You should copy  `MIN(regional_subscription_zone_code.length(), MAX_ZONE_CODE_LEN)`